### PR TITLE
Create cdno.md

### PR DESCRIPTION
**Pull request by LilyAndres**
Edited the cdno.md with the latest information

### CODE REVIEW

File: ontology/cdno.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+layout: ontology_detail
+id: cdno
+title: Crop Dietary Nutrition Ontology
+jobs:
+  - id: https://travis-ci.org/Southern-Cross-Plant-Science/cdno
```
Comment (matentzn) [inline ontology/cdno.md:6]: Does not exist? travis.org was retired afaik!

File: ontology/cdno.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+build:
+  checkout: git clone https://github.com/Southern-Cross-Plant-Science/cdno.git
+  system: git
+  path: "."
+contact:
+  email: l.andres.hernandez.18@student.scu.edu.au
+  label: Liliana Andres Hernandez
+  github: 
+description: CDNO provides structured terminologies to describe nutritional attributes of crop plants and their harvested materials that contribute to human diet. 
+domain: dietary nutritional components, nutritional component concentrations
```
Comment (matentzn) [inline ontology/cdno.md:17]: Please use `diet, metabolomics and nutrition` -> we try to converge all nutrition related ontologies there.

File: ontology/cdno.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- id: chebi
+- id: po
+- id: envo
+
+tracker: https://github.com/Southern-Cross-Plant-Science/cdno/issues
+license:
+  url: https://creativecommons.org/licenses/by/1.0/
+  label: CC-BY
+activity_status: active
+---
```
Comment (matentzn) [inline ontology/cdno.md:37]: Quality control suggested the following errors in this file:

```
FAILURES:
ERROR:ontology/cdno.md: 15:10: trailing spaces (trailing-spaces)
ERROR:ontology/cdno.md: 16:163: trailing spaces (trailing-spaces)
ERROR:ontology/cdno.md: 25:1: wrong indentation: expected 2 but found 0 (indentation)
```

Please fix them all first!

**Review by matentzn — CHANGES_REQUESTED** [review]

File: ontology/cdno.md (hunk 1 above)
Comment (LilyAndres) [inline ontology/cdno.md:6]: Sorry, these two lines should be removed.

File: ontology/cdno.md (hunk 2 above)
Comment (LilyAndres) [reply ontology/cdno.md:17]: Ok, thanks, I will change this.

File: ontology/cdno.md (hunk 3 above)
Comment (LilyAndres) [reply ontology/cdno.md:37]: Thanks, sorry. I will find and remove the spaces.

**Review by LilyAndres — COMMENTED** [review]
Thanks, I will address the issues.

File: ontology/cdno.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+layout: ontology_detail
+id: cdno
+title: Crop Dietary Nutrition Ontology
+build:
+  checkout: git clone https://github.com/Southern-Cross-Plant-Science/cdno.git
+  system: git
+  path: "."
+contact:
+  email: l.andres.hernandez.18@student.scu.edu.au
+  label: Liliana Andres Hernandez
```
Comment (jamesaoverton) [inline ontology/cdno.md:11]: Please include your GitHub user name here.

```
contact:
  email: l.andres.hernandez.18@student.scu.edu.au
  label: Liliana Andres Hernandez
  github: LilyAndres
```

**Review by jamesaoverton — CHANGES_REQUESTED** [review]

File: ontology/cdno.md (hunk 4 above)
Comment (LilyAndres) [reply ontology/cdno.md:11]: Ok, sure thing. I added this info and changed the license to 3.0

**Comment by LilyAndres** [comment]
Thanks @jamesaoverton I added my GitHub name and changed the license in the md file, I committed the changes in the branch. I hope it's better now.

**Review by matentzn — APPROVED** [review]

**Review by jamesaoverton — APPROVED** [review]

**Pull request merged**